microcloud/mdns/lookup.go

```diff
@@ -148,6 +148,10 @@ func Lookup(ctx context.Context, iface *net.Interface, service string, size int)
 	params.Interface = iface
 	params.Entries = entriesCh
 	params.Timeout = 100 * time.Millisecond
+	if !ipv6Supported {
+		params.DisableIPv6 = true
```

Comment:
Would it be worthwhile to check the other way (ipv4) as well? If a system is lacking IPv4 addresses that are eligible for mDNS, would it still result in a panic?

Reply:
fixed